examples/get_recipes_from_minecraft_data.py

```diff
@@ -0,0 +1,64 @@
+import math
```

Comment:
is this file a copy of  experiments/get_recipes_from_minecraft_data.py  or is it simlink?

Reply:
Simlink